lib/audited/auditor.rb

```diff
@@ -220,30 +222,21 @@ def write_audit(attrs)
       end
 
       def presence_of_audit_comment
-        case
-        when !auditing_enabled
```

Comment:
This method was misleading, and again, I think it was trying to do too much / wasn't doing what it said it was.